android/sdk/src/main/java/com/taobao/weex/ui/WXRenderManager.java

```diff
@@ -81,6 +92,54 @@ public WXSDKInstance getWXSDKInstance(String instanceId) {
     return statement.getWXSDKInstance();
   }
 
+    public static int getOpenGLRenderLimitValue() {
+      if(mOpenGLRenderLimitValue == 0){
+          int maxsize = 0;
+              try {
+                  EGL10 egl = (EGL10) EGLContext.getEGL();
+                  EGLDisplay dpy = egl.eglGetDisplay(EGL10.EGL_DEFAULT_DISPLAY);
+                  int[] vers = new int[2];
+                  egl.eglInitialize(dpy, vers);
+                  int[] configAttr = {
+                          EGL10.EGL_COLOR_BUFFER_TYPE, EGL10.EGL_RGB_BUFFER,
+                          EGL10.EGL_LEVEL, 0,
+                          EGL10.EGL_SURFACE_TYPE, EGL10.EGL_PBUFFER_BIT,
+                          EGL10.EGL_NONE};
+                  EGLConfig[] configs = new EGLConfig[1];
+                  int[] numConfig = new int[1];
+                  egl.eglChooseConfig(dpy, configAttr, configs, 1, numConfig);
+                  if(numConfig[0] == 0){
+                      //There is something wrong with opengl environment.
+                      mOpenGLRenderLimitValue = -1;
+                  }
+                      EGLConfig config = configs[0];
```

Comment:
if(numConfig[0] == 0){
                      //There is something wrong with opengl environment.
                      mOpenGLRenderLimitValue = -1;
                     //Destroy EGLDisplay
                    egl.eglTerminate(dpy);
                  }
                  else{
                         //Get max layer size
                  }